app/views/workflows/edit.html.erb

```diff
@@ -78,7 +77,6 @@ See doc/COPYRIGHT.rdoc for more details.
           <%= render partial: 'form',
                      locals: { name: 'author', workflows: @workflows['author'] } %>
         </div>
-      <%= javascript_tag "hideFieldset($('author_workflows'))" unless @workflows['author'].present? %>
```

Comment:
It is not intentional. The fieldset handling _really_ bugs me, since `collapsed` will not suffice to provide the initial hidden state. collapsed is simply a styling for the fieldset itself.

I'll push a fix commit for the sake of this PR, but I'd really like to refactor the toggleable fieldsets into a directive...